main/config/settings/.defaults.yml

```diff
@@ -237,3 +237,6 @@ awsPartition: aws
 # China Mainland regions: 'amazonaws.com.cn'
 # Global regions: 'amazonaws.com'
 awsSuffix: amazonaws.com
+
+# Regions that do not support lambda edge
+disableLambdaEdgeRegions: '["cn-north-1", "cn-northwest-1"]'
```

Comment:
CloudFront and Lambda@Edge are global service. might use partition precise?

Reply:
I suggest we use "disableLambdaEdgeRegions" just for Lambda@Edge, so we can support the regions which partition is "aws" but it does not need or support Lambda@Edge.
The "disableLambdaEdgeRegions" config is in .default.xml file, do not need customer to set. We can maintain this config.

Reply:
> we can support the regions which partition is "aws" but it does not need or support Lambda@Edge

any example? 

Lambda@Edge functions must be deployed to `us-east-1`, same as the certificate of CloudFront. That's why it's a global service not regional service.

Reply:
Use region to check whether edge should be skip or not